source/config/jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
         '\\.(s?css)$': 'identity-obj-proxy',
         '\\.svg$': 'identity-obj-proxy',
     },
-    testURL: 'http://localhost',
+    testURL: 'http://127.0.0.1/',
```

Comment:
out of curiosity - why this change?

Reply:
I was debugging why the tests were failing and I didn't change it back. Fixed